src/app/jscaip/tests/HexagonalGameState.spec.ts

```diff
@@ -153,25 +168,39 @@ describe('HexagonalGameState', () => {
     describe('getEntrance', () => {
         const state: TestingHexagonalState = TestingHexagonalState.empty(7, 7, [3, 2, 1], 0);
         it('should return the correct entrance for lines with a constant q', () => {
-            const line1: HexaLine = HexaLine.constantQ(0);
-            expect(state.getEntranceOnLine(line1).equals(new Coord(0, 3))).toBeTrue();
+            const line: HexaLine = HexaLine.constantQ(0);
+            expect(state.getEntranceOnLine(line).equals(new Coord(0, 3))).toBeTrue();
 
-            const line2: HexaLine = HexaLine.constantQ(4);
-            expect(state.getEntranceOnLine(line2).equals(new Coord(4, 0))).toBeTrue();
+            const otherLine: HexaLine = HexaLine.constantQ(4);
+            expect(state.getEntranceOnLine(otherLine).equals(new Coord(4, 0))).toBeTrue();
         });
         it('should return the correct entrance for lines with a constant r', () => {
-            const line1: HexaLine = HexaLine.constantR(2);
-            expect(state.getEntranceOnLine(line1).equals(new Coord(1, 2))).toBeTrue();
+            const line: HexaLine = HexaLine.constantR(2);
+            expect(state.getEntranceOnLine(line).equals(new Coord(1, 2))).toBeTrue();
 
-            const line2: HexaLine = HexaLine.constantR(4);
-            expect(state.getEntranceOnLine(line2).equals(new Coord(0, 4))).toBeTrue();
+            const otherLine: HexaLine = HexaLine.constantR(4);
+            expect(state.getEntranceOnLine(otherLine).equals(new Coord(0, 4))).toBeTrue();
         });
         it('should return the correct entrance for lines with a constant s', () => {
-            const line1: HexaLine = HexaLine.constantS(4);
-            expect(state.getEntranceOnLine(line1).equals(new Coord(4, 0))).toBeTrue();
+            const line: HexaLine = HexaLine.constantS(4);
```

Comment:
and you get a GWT, and you get a GWT, and you (tu vois le même?)

Reply:
pas convaincu du GWT qui prends 3 lignes sur 2 lignes de test clairs, mais fait !